modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring namespace-scoped Argo Rollouts installation
+
+The following example provides the steps to configure namespace-scoped Argo Rollouts installation.
```

Comment:
```suggestion
The following example provides the steps to configure a namespace-scoped Argo Rollouts installation.
```

Reply:
Consider revising this line. Is the purpose of this task to show an example, or is it to provide the steps to install a namespace-scoped instance of Argo Rollouts? What goal is the customer trying to reach by reading this content?

Reply:
The intention is to provide the steps to install a namespace-scoped instance of Argo Rollouts. Changing the line to as follows: To install a namespace-scoped instance of Argo Rollouts, complete the following steps: